src/config.rs

```diff
@@ -546,3 +554,17 @@ fn get_default_log_level(ini: &Ini) -> Result<log::LevelFilter, Error> {
 
     Ok(log::LevelFilter::Warn)
 }
+
+/// Get the default VCS remote.
+///
+/// To be backward compatible the default remote is still
+/// origin.
+fn get_default_vcs_remote(ini: &Ini) -> String {
+    if let Ok(remote) = env::var("SENTRY_VCS_REMOTE") {
+        return remote.to_string();
```

Comment:
Remove `return` statements to unify with the rest of the file

Reply:
`return`s have been removed.